src/plugins/opensearch_dashboards_overview/public/plugin.ts

```diff
@@ -77,13 +77,14 @@ export class OpenSearchDashboardsOverviewPlugin
         return hasOpenSearchDashboardsApp;
       }),
       distinct(),
-      map((hasOpenSearchDashboardsApp) => {
+      map(() => {
         return () => {
-          if (!hasOpenSearchDashboardsApp) {
-            return { status: AppStatus.inaccessible, navLinkStatus: AppNavLinkStatus.hidden };
-          } else {
-            return { status: AppStatus.accessible, navLinkStatus: AppNavLinkStatus.default };
-          }
+          // Wazuh: Hide the OpenSearch Dashboards Overview app
+          // TODO (unknown reason: `Add your data` button ?? )
+          return {
+            status: AppStatus.inaccessible,
+            navLinkStatus: AppNavLinkStatus.hidden,
+          };
```

Comment:
Hides the OpenSearch Dashboards Overview app

Would be better if done in a separate issue.

Also, there are **links** to this app that have not been disabled, causing a 404 in the app. For example, the card at `Home` links to `/app/opensearch_dashboards_overview`:
![image](https://user-images.githubusercontent.com/15186973/226687703-815b0735-2f32-4193-839a-8abd6b921c55.png)

Reply:
Done